kore/test/Test/Kore/Simplify/Pattern.hs

```diff
@@ -54,10 +54,7 @@ test_Pattern_simplify =
                     (Mock.constr10 fOfX)
                     ( makeAndPredicate
                         (makeCeilPredicate fOfX)
-                        ( makeExistsPredicate
-                            Mock.yConfig
-                            (makeCeilPredicate fOfY)
-                        )
+                        (makeCeilPredicate fOfY)
```

Comment:
Reminder about what we discussed during the meeting, to modify the test names to include what's happening with the exists there.